.github/dependabot.yml

```diff
@@ -1,5 +1,83 @@
 version: 2
 updates:
+  - package-ecosystem: npm
+    directory: "/web/packages/design"
+    schedule:
+      interval: monthly
+      day: "sunday"
+      time: "09:00" # 9am UTC
+    open-pull-requests-limit: 10
+    reviewers:
+      - avatus
+      - kimlisa
+      - rudream
+      - bl-nero
+    labels:
+      - "dependencies"
+      - "ui"
+      - "no-changelog"
+  - package-ecosystem: npm
+    directory: "/web/packages/build"
```

Comment:
nit: Declare packages in alphabetical/directory order? Makes it easier to find whatever we might look for.